types/svg-maps__brazil/index.d.ts

```diff
@@ -0,0 +1,22 @@
+// Type definitions for @svg-maps/brazil 1.0
+// Project: https://github.com/VictorCazanave/svg-maps/tree/master/packages/brazil
+// Definitions by: Joao Sutel <https://github.com/joaosutel>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// tslint:disable-next-line no-single-declare-module
```

Comment:
Please write this using the standard module format.

Reply:
Done